src/plugins/recordTypes/uoc/fields.js

```diff
@@ -0,0 +1,290 @@
+              },
+            },
+          },
+        },
+        authorizationDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.authorizationDate.name',
+                defaultMessage: 'Authorization date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        authorizationNote: {
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.authorizationNote.name',
+                defaultMessage: 'Authorization note',
+              },
+            }),
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        startSingleDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.startSingleDate.name',
+                defaultMessage: 'Start/single date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        endDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.endDate.name',
+                defaultMessage: 'End date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        userGroupList: {
+          [config]: {
+            view: {
+              type: CompoundInput,
+            },
+          },
+          userGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.uoc_common.userGroup.name',
+                  defaultMessage: 'User',
+                },
+              }),
+              repeating: true,
+              view: {
+                type: CompoundInput,
+                props: {
+                  tabular: true,
+                },
+              },
+            },
+            user: {
+              [config]: {
+                messages: defineMessages({
+                  name: {
+                    id: 'field.uoc_common.user.name',
+                    defaultMessage: 'Name',
+                  },
+                }),
+                view: {
+                  type: AutocompleteInput,
+                  props: {
+                    source: 'person/local,person/shared,organization/local,organization/shared',
+                  },
+                },
+              },
+            },
+            userType: {
+              [config]: {
+                messages: defineMessages({
+                  name: {
+                    id: 'field.uoc_common.userType.name',
+                    defaultMessage: 'User type',
+                  },
```

Comment:
The name message should just be "Type", and there should be a fullName message of "User type".